tools/update-readme.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * @fileoverview Script to update the README with team and sponsors.
+ * Note that this requires eslint/website to be available in the same
+ * directory as the eslint repo.
+ *
+ *   node tools/update-readme.js
+ *
+ * @author Harish Kumar S S
+ */
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+import { readFileSync, writeFileSync } from "node:fs";
+import { stripIndents } from "common-tags";
+import ejs from "ejs";
+import got from "got";
+
+//-----------------------------------------------------------------------------
+// Data
+//-----------------------------------------------------------------------------
+
+const SPONSORS_URL =
+	"https://raw.githubusercontent.com/eslint/eslint.org/main/src/_data/sponsors.json";
+const TEAM_URL =
+	"https://raw.githubusercontent.com/eslint/eslint.org/main/src/_data/team.json";
+const README_FILE_PATHS = [
```

Comment:
Can we autogenerate this so we don't need to keep updating?

I think we can hardcode `"./README.md"` and then just add every directory under `./packages`.

Reply:
Done